dbms/src/TiDB/Schema/TiDB.cpp

```diff
@@ -375,7 +377,8 @@ try
     json->set("origin_default", origin_default_value);
```

Comment:
`default`, `origin_default` and `default_bit` are also useless?

Reply:
Also removed